samples/get-entity-type-sample.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Gets details of a single EntityType.
+ * See https://cloud.google.com/vertex-ai/docs/featurestore/setup before running
+ * the code snippet
+ */
+
+'use strict';
+
+async function main(
+  project,
+  featurestoreId,
+  entityTypeId,
+  location = 'us-central1',
+  apiEndpoint = 'us-central1-aiplatform.googleapis.com',
+  timeout = 300000
```

Comment:
Can you change to 5s for Get/List methods

Reply:
Made the respective changes